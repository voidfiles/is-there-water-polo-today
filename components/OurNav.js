import Image from "next/image";

export default function OurNav() {
  return (
    <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
      <div className="flex justify-between h-16">
        <div className="flex">
          <div className="flex-shrink-0 flex items-center">
            <div className="block h-8 w-auto">
              <Image
                src="/waterpolo.png"
                alt="Water Polo"
                width="32"
                height="32"
              />
            </div>
            <span>Is there water polo today?</span>
          </div>
        </div>
      </div>
    </div>
  );
}
