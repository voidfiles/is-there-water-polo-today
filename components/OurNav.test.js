import React from "react";
import renderer from "react-test-renderer";

import OurNav from "./OurNav";

it("renders a nav", () => {
  const tree = renderer.create(<OurNav />).toJSON();
  expect(tree).toMatchSnapshot();
});
