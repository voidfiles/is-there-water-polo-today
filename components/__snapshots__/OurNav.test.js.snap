// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a nav 1`] = `
<div
  className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8"
>
  <div
    className="flex justify-between h-16"
  >
    <div
      className="flex"
    >
      <div
        className="flex-shrink-0 flex items-center"
      >
        <div
          className="block h-8 w-auto"
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="Water Polo"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="Water Polo"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fwaterpolo.png&w=64&q=75"
                srcSet="/_next/image?url=%2Fwaterpolo.png&w=32&q=75 1x, /_next/image?url=%2Fwaterpolo.png&w=64&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
        <span>
          Is there water polo today?
        </span>
      </div>
    </div>
  </div>
</div>
`;
